_posts/2020-04-21-sparklyr-1.2.0-released/sparklyr-1.2.0.Rmd

```diff
@@ -0,0 +1,42 @@
+---
+title: "Sparklyr v1.2.0 released"
+description: |
+  Sparklyr v1.2.0 is now available. This release features new functionalities such as support for Databricks Connect and `registerDoSpark`, inter-op improvements for working with Spark-3.0.0-preview, as well as a number of bug fixes and improvements addressing user-visible pain points.
+author:
+  - name: Yitao Li
+    url: https://github.com/yl790
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com
+date: 04-21-2020
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+Behold the glory that is Sparklyr v1.2.0!
+
+In this release, the following new hotnesses have emerged into spotlight:
+
+- Support for Databricks Connect [https://docs.databricks.com/dev-tools/databricks-connect.html], allowing Sparklyr to connect to a Databircks cluster. This is a result of close collaboration between Javier from RStudio and Hossein, Andy, et al from DataBricks.
+- A `registerDoSpark` method to create foreach parallel backend powered by Spark
```

Comment:
if that's correct content-wise, I'd write 

a `foreach` parallel backend

Reply:
Yep I think foreach in back quotes is better for readability :+1: